Gems/Atom/Bootstrap/Code/Source/BootstrapSystemComponent.cpp

```diff
@@ -74,6 +74,45 @@ void cvar_r_renderPipelinePath_Changed(const AZ::CVarFixedString& newPipelinePat
     }
 }
 
+void cvar_r_antiAliasing_Changed(const AZ::CVarFixedString& newAtiAliasing)
+{
+    auto viewportContextManager = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
+    if (!viewportContextManager)
+    {
+        return;
+    }
+    auto viewportContext = viewportContextManager->GetDefaultViewportContext();
+    if (!viewportContext)
+    {
+        return;
+    }
+
+    AZ::Render::Bootstrap::RequestBus::Broadcast(&AZ::Render::Bootstrap::RequestBus::Events::SwitchAntiAliasing, newAtiAliasing.c_str(), viewportContext);
+}
+
+void cvar_r_multiSample_Changed(const uint16_t& newSampleCount)
+{
+    auto viewportContextManager = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
+    if (!viewportContextManager)
+    {
+        return;
+    }
+    auto viewportContext = viewportContextManager->GetDefaultViewportContext();
+    if (!viewportContext)
+    {
+        return;
+    }
+    if (newSampleCount > 0 && ((newSampleCount & (newSampleCount - 1))) == 0)
```

Comment:
Add a comment about (newSampleCount & (newSampleCount - 1)) to ensure it is to check if the value is a power of 2. We should also probably clamp the value to be 16 for max value.

Reply:
I clamp it up to 8, it seems that 16 is not acceptable by design [here](https://github.com/o3de/o3de/blob/66279081935e211510585a2fa725f1c2cc771cbb/Gems/Atom/Feature/Common/Code/Source/CoreLights/LightCullingTilePreparePass.cpp#L106)